pkg/kubelet/util/pluginwatcher/BUILD

```diff
@@ -6,17 +6,16 @@ go_library(
         "example_handler.go",
         "example_plugin.go",
         "plugin_watcher.go",
```

Comment:
Let's go ahead and move this entire package from `pkg/kubelet/util/pluginwatcher/*` to `pkg/kubelet/pluginmanager/pluginwatcher/*` so that all the plugin code is in one place.